### DISCUSSION TITLE
提示/bin/sh: nohup: not found,但是已经装过依赖了。

### DISCUSSION THREAD

**Discussion by pst916916** [vernesong/OpenClash#3142] [Q&A]
2023-03-26 00:24:03 OpenClash 开始关闭...
2023-03-26 00:24:02 错误：无法获取 General 部分的转发端口设置, OpenClash 使用原始配置文件启动失败
2023-03-26 00:24:02 错误：无法获取 DNS 部分的运行模式设置, OpenClash 使用原始配置文件启动失败
2023-03-26 00:24:02 错误：无法获取 DNS 部分的监听端口设置, OpenClash 使用原始配置文件启动失败
2023-03-26 00:24:01 错误：OpenClash 启动失败，尝试使用原始配置文件启动...
2023-03-26 00:24:00 第六步: 等待主程序下载外部文件...
2023-03-26 00:23:57 第五步: 检查内核启动状态...
/bin/sh: nohup: not found
2023-03-26 00:23:56 提示：检测到配置了 TUN 内核专属功能，调用 TUN 内核启动...
2023-03-26 00:23:56 第四步: 启动主程序...
2023-03-26 00:23:56 提示：开始运行自定义覆写脚本...
2023-03-26 00:23:55 提示：检测到DNS选项下的 Nameserver 未设置服务器，开始补全...

现实没有找到nohup，但是我十分确定，已经装过了.
![QQ截图20230326010418](https://user-images.githubusercontent.com/32671428/227731392-d4afe42f-2dd5-48f6-b095-322d5964e510.png)
这是软件包里的信息。

OpenClash 调试日志

生成时间: 2023-03-26 00:38:41
插件版本: 
隐私提示: 上传此日志前请注意检查、屏蔽公网IP、节点、密码等相关敏感信息

```

#===================== 系统信息 =====================#

主机型号: Xiaomi Mi Router AX9000 IPQ807x/AP-HK14
固件版本: 
LuCI版本: git-20.356.64372-1259bb1-1
内核版本: 4.4.60
处理器架构: aarch64_cortex-a53+crypto

#此项有值时,如不使用IPv6,建议到网络-接口-lan的设置中禁用IPV6的DHCP
IPV6-DHCP: server

DNS劫持: Dnsmasq 转发
#DNS劫持为Dnsmasq时，此项结果应仅有配置文件的DNS监听地址
Dnsmasq转发设置: 

#===================== 依赖检查 =====================#

dnsmasq-full: 已安装
coreutils: 未安装
coreutils-nohup: 未安装
bash: 已安装
curl: 已安装
ca-certificates: 已安装
ipset: 已安装
ip-full: 已安装
libcap: 已安装
libcap-bin: 已安装
ruby: 已安装
ruby-yaml: 已安装
ruby-psych: 已安装
ruby-pstore: 已安装
kmod-tun(TUN模式): 已安装
luci-compat(Luci >= 19.07): 已安装
kmod-inet-diag(PROCESS-NAME): 未安装
unzip: 已安装
iptables-mod-tproxy: 已安装
kmod-ipt-tproxy: 已安装
iptables-mod-extra: 已安装
kmod-ipt-extra: 已安装
kmod-ipt-nat: 已安装

#===================== 内核检查 =====================#

运行状态: 未运行
已选择的架构: TUN

#下方无法显示内核版本号时请确认您的内核版本是否正确或者有无权限
Tun内核版本: 2023.03.04-5-g4a8cefb
Tun内核文件: 存在
Tun内核运行权限: 正常

Dev内核版本: v1.13.0-7-g4ffc999
Dev内核文件: 存在
Dev内核运行权限: 正常

Meta内核版本: alpha-g2f992e9
Meta内核文件: 存在
Meta内核运行权限: 正常

#===================== 插件设置 =====================#

当前配置文件: /etc/openclash/config/Dler Cloud - smart.yaml
启动配置文件: /etc/openclash/Dler Cloud - smart.yaml
运行模式: redir-host-mix
默认代理模式: rule
UDP流量转发(tproxy): 停用
自定义DNS: 停用
IPV6代理: 停用
IPV6-DNS解析: 停用
禁用Dnsmasq缓存: 启用
自定义规则: 停用
仅允许内网: 启用
仅代理命中规则流量: 停用
仅允许常用端口流量: 停用
绕过中国大陆IP: 停用
路由本机代理: 启用

#启动异常时建议关闭此项后重试
混合节点: 停用
保留配置: 停用

#启动异常时建议关闭此项后重试
第三方规则: 停用

**Comment by shaokn** [comment]
我也遇到这个问题了。重新安装一下nohup就好了。

`opkg install coreutils-nohup --force-reinstall`

**Reply by LyndonWu233** [reply]
十分感谢，感谢您的指导，帮助我解决问题。祝您新年快乐，阖家安康！

**Reply by dexhunter** [reply]
不知道为什么每次更新订阅配置的时候都会报这个错，然后重新安装一下就好了

**Reply by leeyxq** [reply]
Nice,tks

**Comment by vernesong** [comment]
更新失败才会这样，建议手动更新
